docker/Dockerfile-ubuntu-19.04

```diff
@@ -1,33 +1,11 @@
 FROM ubuntu:disco
 ENV DEBIAN_FRONTEND noninteractive
 COPY build-and-install-scafacos.sh /tmp/
-RUN apt-get update && apt-get install -y \
-    apt-utils \
-    cmake \
-    build-essential \
-    pkg-config \
-    openmpi-bin libopenmpi-dev \
-    gfortran \
-    libfftw3-dev \
-    libboost-dev libboost-serialization-dev libboost-mpi-dev libboost-filesystem-dev libboost-test-dev \
-    libgsl-dev \
-    cython3 python3 python3-numpy python3-scipy python3-h5py \
-    python3-setuptools \
-    lcov \
-    git \
-    python3-vtk7 \
-    libhdf5-openmpi-dev \
-    libhdf5-openmpi-103:amd64 \
-    libhdf5-103:amd64 \
-    libblas-dev \
-    liblapack-dev \
-    autoconf \
-    vim \
-    ccache \
-    curl \
-    jq \
-&& apt-get clean \
-&& rm -rf /var/lib/apt/lists/*
+COPY ubuntu-packages.txt /tmp
+RUN apt-get update && xargs -a /tmp/ubuntu-packages.txt apt-get install -y && \
```

Comment:
The older syntax followed the docker best practices by putting the `&&` at the beginning of lines to make it easier to distinguish which line continuations were shell commands (it starts with `&&`) and which lines continuations were list of packages (it doesn't start with `&&`). Why move the `&&` to the end of lines?